apps/www/components/Payment/Form/StripeWalletHelpers.tsx

```diff
@@ -0,0 +1,64 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react'
+
+// Check if the website is running in a secure document.
+function isInSecureWindow(): boolean {
+  return typeof window !== 'undefined' && window.location.protocol === 'https:'
+}
+
+// ---- Apple Pay ----
+
+// Declare ApplePaySession property to prevent TypeScript errors
+declare global {
+  interface Window {
+    ApplePaySession: ApplePaySession | undefined
+  }
+}
+
+function isApplePayAvailable(): boolean {
+  if (!window.ApplePaySession) {
```

Comment:
don't forget to add it here ;-)
```suggestion
  if (!isInSecureWindow() && !window.ApplePaySession) {
```

Reply:
Shouldn't be an issue since ApplePaySession isn't defined on non HTTPS. But "doppelt hält besser" xD